frontend/src/layouts/new-layout.js

```diff
@@ -347,6 +362,17 @@ const NewLayout = ({
         )}
       </div>
       <Login visible={loginVisible} close={() => setLoginVisible(false)} />
+      {/* iframe ROCKETCHAT */}
+      <iframe
+        src={`${process.env.NEXT_PUBLIC_CHAT_API_DOMAIN_URL}/home?standalone=true&layout=embedded`}
```

Comment:
@ifirmawan why do we have the `standalone=true` here?

Reply:
Hi @lucassousaf, ah I missed removing it. at that time I want to make sure the user session between 2 iframes